project/Dependencies.scala

```diff
@@ -17,19 +17,17 @@ object Dependencies {
   val sslConfigVersion = "0.2.1"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.11.8"), // "2.12.0-RC2"
+    crossScalaVersions := Seq("2.11.8"), // "2.12.0"
```

Comment:
@johanandren isn't it time that we add `2.12.0` to the `Seq` and actually cross publish all releases automatically from now? Is there still some dependencies that doesn't exist for 2.12? Can we define a different `crossScalaVersions` for those projects?